source/dpp/translation/aggregate.d

```diff
@@ -421,6 +421,18 @@ string[] translateBitField(in from!"clang".Cursor cursor,
     return [text("    ", type, `, "`, spelling, `", `, cursor.bitWidth, `,`)];
 }
 
+private from!"clang".Type pointeeTypeFor(in from!"clang".Type type)
+    @safe
+{
+    import clang: Type;
+
+    Type pointeeType = type.pointee.canonical;
+    while (pointeeType.kind == Type.Kind.Pointer)
+        pointeeType = pointeeType.pointee.canonical;
```

Comment:
Shouldn't this recurse instead? Triple pointers are uncommon, but I see no reason to artifically limit this.

Reply:
Shouldn't this work for any number of pointers? This while-loop moves to the pointee type until it gets to a type that is no longer a pointer. I've also added a unit test with a triple pointer, but I guess it should work with any given number.

Reply:
You're right, I was confused.